lib/VideoPlayer/VideoPlayer.less

```diff
@@ -45,32 +45,32 @@
 
 /* Fullscreen CSS */
 :-webkit-full-screen.moon-video-player {
-	.full-screen-video-player;
+	.full-screen-video-player();
```

Comment:
This was actually not needed and superseded by https://github.com/enyojs/moonstone/pull/2306. It only appears this way because we split up the commits and moved one of the commits to the other PR.